TombRaiderIII/Autosplitter.cs

```diff
@@ -74,6 +74,14 @@ internal class Autosplitter : IAutoSplitter, IDisposable
             if (currentLevelTicks - oldLevelTicks == 0)
                 return null;
 
+            // TR3's IGT ticks during globe level selection; the saved end-level IGT is unaffected, thus the overall FG IGT is also unaffected.
+            // If a runner is watching LS's IGT, this may confuse them, despite it being a non-issue for the level/FG IGT.
+            // To prevent the ticks from showing in LS, we use the fact that LevelComplete isn't reset to 0 until the next level is loaded.
+            Level currentLevel = GameMemory.Data.Level.Current;
+            bool levelComplete = GameMemory.Data.LevelComplete.Current;
+            if (_completedLevels.Contains(currentLevel) && levelComplete)
+                return null;
```

Comment:
This still has this odd thing with the timer stopping at the stats screen, then IGT advancing a tick during the next level load. The following code fixes it for me (had some trouble coming up with a meaningful var name):

```cs
            Level currentLevel = GameMemory.Data.Level.Current;
            bool oldLevelComplete = GameMemory.Data.LevelComplete.Old;
            bool currentLevelComplete = GameMemory.Data.LevelComplete.Current;
            bool reallyCompleteLevel = oldLevelComplete == currentLevelComplete == true;
            if (_completedLevels.Contains(currentLevel) && reallyCompleteLevel)
                return null;
```

Reply:
Hmm. Perhaps `levelStillComplete`?

Reply:
I'm okay with that.